config.json

```diff
@@ -0,0 +1,20 @@
+{
+    "popslider_min": 750,
+    "popslider_max": 1000000,
+    "popslider_start": [50000, 1000000],
+    "popslider_range": {
+        "min": [750, 250],
+        "5%": [1000, 1000],
+        "30%": [10000, 5000],
+        "50%": [25000, 25000],
+        "70%": [100000, 100000],
+        "max": 1000000
+    },
+    "min_strength": 250,
+    "loglevel": "WARN",
+    "label_threshold": 0.4,
+    "rel_categories": ["commuting", "shopping", "leisure", "moving", "education", "collaboration", "transportation", "other"],
```

Comment:
beter deze units in DB gooien aangezien ze in backend ook worden gebruikt. Vervolgens on load inladen?